src/ApiService/ApiService/onefuzzlib/Events.cs

```diff
@@ -47,8 +48,26 @@ public class Events : IEvents {
         }
 
         public virtual async Async.Task QueueSignalrEvent(DownloadableEventMessage message) {
-            var ev = new SignalREvent("events", new List<DownloadableEventMessage>() { message });
-            await _queue.SendMessage("signalr-events", JsonSerializer.Serialize(ev, _options), StorageType.Config);
+            var tags = new (string, string)[] {
+                ("event_type", message.EventType.ToString()),
+                ("event_id", message.EventId.ToString())
+            };
+            try {
+                var ev = new SignalREvent("events", new List<DownloadableEventMessage>() { message });
+                await _queue.SendMessage("signalr-events", JsonSerializer.Serialize(ev, _options), StorageType.Config);
```

Comment:
We should centralize the logic of truncating the message to SendMessage (and/or QueueObject) to avoid repetition. We already use a similar logic in AddEvent.